### PR TITLE
Migrate to girder3

### PR DESCRIPTION
**Pull request by mgrauer**
@zachmullen Does this look sensible?

@jbeezley helped me get the client build working, and I've tested that the functionality works on a local install.

### CODE REVIEW

File: girder_monkeybrains/__init__.py [hunk 1]
```diff
@@ -24,8 +24,11 @@
 from girder.api.rest import RestException
 from girder.api.describe import Description
 from girder.constants import AccessType
+from girder.plugin import getPlugin, GirderPlugin
 from girder.utility.model_importer import ModelImporter
 
+
+
```
Comment (zachmullen) [inline girder_monkeybrains/__init__.py:31]: Might want to PEP8 this

File: girder_monkeybrains/__init__.py [hunk 2]
```diff
@@ -41,9 +44,8 @@ def setCollectionMonkeybrains(self, collection, params):
             del collection[MONKEYBRAINS_FIELD]
         else:
             collection[MONKEYBRAINS_FIELD] = monkeybrains
-        self.model('collection').save(collection, validate=False)
-        return self.model('collection').filter(collection,
-                                               self.getCurrentUser())
+        ModelImporter.model('collection').save(collection, validate=False)
+        return ModelImporter.model('collection').filter(collection, self.getCurrentUser())
```
Comment (zachmullen) [inline girder_monkeybrains/__init__.py]: We just import these models statically now, i.e.

```python
from girder.models.collection import Collection


Collection().save(...)
```

Also the call to `filter` should be replace with the `filtermodel` decorator.

**Review by zachmullen — CHANGES_REQUESTED** [review]

File: girder_monkeybrains/__init__.py (hunk 2 above)
Comment (zachmullen) [reply girder_monkeybrains/__init__.py]: Normatively: if the model you want to load is known statically, use the static instantiation.

File: girder_monkeybrains/web_client/package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "@girder/monkeybrains",
+    "version": "1.0.0",
+    "peerDepencencies": {
+        "@girder/core": "*"
```
Comment (brianhelba) [inline girder_monkeybrains/web_client/package.json:5]: This should use version `^3`. There are v2 releases in that repo (and could possibly be more in the future).

Comment (mgrauer) [reply girder_monkeybrains/web_client/package.json:5]: I'm not sure what you are saying.

    "@girder/core": "^3"

?

This frontend stuff is pure cargo cult for me.

Comment (jbeezley) [reply girder_monkeybrains/web_client/package.json:5]: IMO trying to specify versions for peer dependencies causes more harm then good even when they are applied correctly.  For this reason, I didn't bother with it in the core plugins or the migration docs.

**Comment by mgrauer** [comment]
@zachmullen PTAL.

**Review by zachmullen — APPROVED** [review]

**Pull request merged**